core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -67,7 +67,6 @@
 import jenkins.util.io.OnMaster;
 import net.sf.json.JSONObject;
 
-import org.acegisecurity.Authentication;
```

Comment:
noting regression https://issues.jenkins-ci.org/browse/JENKINS-64746 addressed in #5216